### ISSUE TITLE
Bakkesmod can't detect rocket league if launched with flatpak protontricks

### ISSUE THREAD

**Issue by maroc81** [CrumblyLiquid/BakkesLinux#12]
I followed this excellent guide to setup Bakkesmod.  I used the flatpak version of protontricks to install and then launch Bakkesmod. Everything appeared to work great but due to flatpak sandboxing, Bakkesmod can't detect the rocket league process to do the injection.

I figured out how to launch Bakkesmod manually using proton and that worked.  I also realized that your bakkes.sh script works since it doesn't use protontricks.

So to summarize:
 - You can use the flatpak version of protontricks to install Bakkesmod
 - Use the bakkes.sh script after install to launch

You may want to add this note somewhere in your instructions.

Thanks for doing all the work to put these instructions together! I gave up trying to get Bakkesmod working about a year ago and then I found these instructions.

**Comment by CrumblyLiquid** [comment]
Thanks! I've added a note to TROUBLESHOOTING.md and also a [!TIP] block linking to it in the README.md and again thanks for the info!
